app/main.py

```diff
@@ -1 +1,8 @@
-# write your code here
+def copy_file(command: str) -> None:
+    command_list = command.split(" ")
```

Comment:
you can use split() without specifying a delimiter - it will automatically split by whitespace